cornflow/endpoints/execution.py

```diff
@@ -159,11 +165,24 @@ def get(self, idx):
         and an integer with the HTTP status code.
         :rtype: Tuple(dict, integer)
         """
-        self.user_id, self.admin, self.super_admin = Auth.return_user_info(request)
-        status = ExecutionModel.get_one_execution_from_user(self.user_id, idx).finished
-        # TODO: call airflow to check status
-        if not status:
-            # Here we should call airflow to check solving status
-            pass
+        airflow_conf = dict(url=current_app.config['AIRFLOW_URL'],
+                            user=current_app.config['AIRFLOW_USER'],
+                            pwd=current_app.config['AIRFLOW_PWD'])
 
-        return {'finished': status}, 200
+        self.user_id, self.admin, self.super_admin = Auth.return_user_info(request)
+        execution = ExecutionModel.get_one_execution_from_user(self.user_id, idx)
```

Comment:
Maybe the status variable is not needed. execution.finished is by defect false, so you could do if execution.finished instead

Reply:
Yes, I will change this. I think we returned the `status` boolean variable before and I did not change it.